packages/test-interface-criteria/built-types/types/Criteria.d.ts

```diff
@@ -5,7 +5,7 @@ import { TestDataShape } from './TestDataShape';
 
 export type OpportunityConstraint = (opportunity: Opportunity, options?: Options) => boolean;
 
-export type OfferConstraint = (offer: Offer, opportunity: Opportunity, options?: Options) => boolean;
+export type OfferConstraint = (offer: Offer, opportunity?: Opportunity, options?: Options) => boolean;
```

Comment:
@lukehesluke this change seems to create quite significant knock-on effects within the generated types. It looked like it was a typo that needed fixing, but I'm wondering if it was actually intentional?

Would you be able to shed any light on this?

Reply:
@nickevansuk why did this seem like a typo? This was made quite a while ago but it probably was intentional

Reply:
For the record: this was a result of upgrading TS locally. Reverting TS removed the need to make any type changes at all.